Dockerfile

```diff
@@ -13,35 +7,26 @@ RUN apk update \
   && apk add --no-cache openssh-client make git ca-certificates tar curl \
   && update-ca-certificates
 
-RUN curl -L -o glide.zip "$GLIDE_DOWNLOAD_URL"
-RUN unzip glide.zip linux-amd64/glide
-RUN mv linux-amd64/glide /usr/local/bin
-RUN rm -rf linux-amd64 glide.zip
-
-RUN curl -L -o /usr/local/bin/dep "$DEP_DOWNLOAD_URL"
-
 RUN curl -L -o goreleaser_Linux_x86_64.tar.gz "$GO_RELEASER_DOWNLOAD_URL"
 RUN tar -xf goreleaser_Linux_x86_64.tar.gz
 RUN mv goreleaser /usr/local/bin/goreleaser
 
-FROM golang:1.12.5-alpine3.9
+FROM golang:1.13.6-alpine3.11
 LABEL maintainer="Vidsy <tech@vidsy.co>"
 
 ARG VERSION
 LABEL version=$VERSION
 
 ENV CGO_ENABLED 0
 ENV GOCACHE /go/pkg/cache
+ENV GOPRIVATE github.com/vidsy/*
```

Comment:
@krak3n @luk4ward Thoughts on this being in here?

It is a public repo, so obviously this isn't relevant to anyone outside of the Vidsy org.

If not in here we could add it to a context in CircleCI but would have to make sure that context was always applied to each go job.

Reply:
I'm not adverse to this being here, but I think I would prefer a CircleCI context.